performance_benchmarks/servers/cheroot_wsgi.py

```diff
@@ -0,0 +1,17 @@
+"""
+Adapted from https://github.com/cherrypy/cheroot/blob/master/cheroot/test/test_wsgi.py
+"""
+
+from cheroot import wsgi
+
+def simple_wsgi_server():
+    def app(_environ, start_response):
```

Comment:
This could probably just reuse https://github.com/jamesroberts/fastwsgi/blob/main/performance_benchmarks/servers/gunicorn_wsgi.py.

Reply:
Nice catch!